docs/Documentation.md

```diff
@@ -90,10 +90,57 @@ mongo_db_bundle:
       client_name: ~
       database_name: ~
 
+  # Service reference to provide URI options - see example below
+  uriOptions: 'App\Services\UriOptionsProvider' # default null
+  
   # Service reference to provide driver options - see example below
   driverOptions: 'App\Services\DriverOptionsProvider' # default null
 ```
 
+### Uri options
+
+You might need to specify some URI options for constructing the `MongoDB\Client`. Read the [reference] for a complete
```

Comment:
Is the reference link missing?
Are you looking for something like this? https://www.mongodb.com/docs/manual/reference/connection-string/#std-label-connections-connection-options

Reply:
the link refers to https://www.php.net/manual/en/mongodb-driver-manager.construct.php